modules/nf-core/cnvkit/genemetrics/main.nf

```diff
@@ -0,0 +1,40 @@
+process CNVKIT_GENEMETRICS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::cnvkit=0.9.9 bioconda::samtools=1.16.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cnvkit:0.9.9--pyhdfd78af_0':
+        'quay.io/biocontainers/cnvkit:0.9.9--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(cnr)
+    tuple val (meta), path(cns)
+
+    output:
+    tuple val(meta), path("*.tsv"), emit: tsv
+    //tuple val(meta), path("*.cnn"), emit: cnn
```

Comment:
I know this is from upstream, but any idea what is going on here?

Reply:
As If i left it as emitting the `.cnn` it overwirtes some other outputs so I changed it to .tsv

Reply:
You could change the output name using `ext.prefix` and avoid any overwritting.

Reply:
Although it does output a tsv file so makes it easier for a user to know what delimiter to use when loading into r/python